apps/dashr-wind-steaming/Procfile

```diff
@@ -0,0 +1 @@
+web: gunicorn --pythonpath apps/dash-wind-streaming app:server
```

Comment:
@Scott-Clone Please change the `Procfile` so that it is correctly formatted for R apps, e.g.

```
web: R -f /app/apps/"$DASH_APP_NAME"/app.R
```